### ISSUE TITLE
Obtain ChannelID public from link

### ISSUE THREAD

**Issue by dudu631** [gram-js/gramjs#278]
Hi,

I'm trying to obtain the channel id from channel-group link (t.me/channel).

For private channel links with hash at the link, I'm using `Api.messages.CheckChatInvite`, and works fine.

For public channels, I'm currently using `Api.messages.GetAllChats` and iterating over all chats comparing the link last path with `channel.username`. 

Is there a better way of doing it?

**Comment by painor** [comment]
you can do client.getEntity("username")

**Comment by dudu631** [comment]
Yep, that works and it's neat. Couldn't find that info easy on the docs. 

Thank you!
